content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -129,17 +129,28 @@ Plugins wishing to perform "pre-reserve" work should use the
 NormalizeScore extension point.
 {{< /note >}}
 
-### Reserve
-
-This is an informational extension point. Plugins which maintain runtime state
-(aka "stateful plugins") should use this extension point to be notified by the
-scheduler when resources on a node are being reserved for a given Pod. This
-happens before the scheduler actually binds the Pod to the Node, and it exists
-to prevent race conditions while the scheduler waits for the bind to succeed.
-
-This is the last step in a scheduling cycle. Once a Pod is in the reserved
-state, it will either trigger [Unreserve](#unreserve) plugins (on failure) or
-[PostBind](#post-bind) plugins (on success) at the end of the binding cycle.
+### Reserve {#reserve}
+
+A Reserve plugin implements two methods, namely Reserve and Unreserve, to serve
+as informational extension points. Plugins which maintain runtime state (aka
```

Comment:
let's say something like (@Huang-Wei for thoughts):

The Reserve extension point backs two informational scheduling phases: Reserve and Unreserve. Plugins implement two methods, Reserve and Unreserve, that get called during these phases, respectively.

Sorry for the back and forth, but I don't find an easy way to express the fact that Unreserve is not an extension point.

Reply:
it is a callback

Reply:
LGTM, slightly reworded:

```
The Reserve extension point backs two informational scheduling phases: Reserve and Unreserve,
which are implemented in pairs and get called during these phases, respectively.
```

Reply:
Done. I've slightly modified @Huang-Wei's suggestion to remove the respectively part as I don't think it's necessary (as there aren't two pairs of objects like in @alculquicondor's suggestion). PTAL.

Reply:
Also, we can explicitly state that Unreserve is not an extension point in addition to these changes to make it very clear?